### DISCUSSION TITLE
Add loader from npm

### DISCUSSION THREAD

**Discussion by chronosye** [suren-atoyan/monaco-react#392] [Q&A]
Hello!
I am using monaco-editor/react v3.7.5.
I want to ask, how can I implement that loader is not loaded from CDN, but from installed npm package?

**Comment by suren-atoyan** [comment]
@chronosye it would be much easier with the latest version, but if you can't update the package with v3.7.5 you should use [loader-config](https://github.com/suren-atoyan/monaco-react#loader-config) to load monaco from other sources. By default it's something like this:

```js
paths: {
  vs: 'https://cdn.jsdelivr.net/npm/monaco-editor@0.33.0/min/vs',
}
```

you can make it:

```js
paths: {
    vs: '/static/path/to/monaco',
  }
```
